airbyte-webapp/src/components/ui/StepsMenu/StepsMenu.tsx

```diff
@@ -25,17 +24,17 @@ const Content = styled.div`
   font-family: ${({ theme }) => theme.regularFont};
 `;
 
-export const StepsMenu: React.FC<IProps> = ({ data, onSelect, activeStep, lightMode }) => {
+export const StepsMenu: React.FC<StepMenuProps> = ({ data, onSelect, activeStep, lightMode }) => {
   return (
     <Content>
       {data.map((item, key) => (
         <Step
-          status={item.status}
-          isPartialSuccess={item.isPartialSuccess}
+          icon={item.icon}
           lightMode={lightMode}
           key={item.id}
           num={key + 1}
-          {...item}
+          name={item.name}
```

Comment:
:information_source: We spreaded this.. but only for 2 properties, the rest was still explicitally passed on above ... :shrug:  I made them now all explicitally passed through.